.github/workflows/ci.yml

```diff
@@ -49,6 +49,10 @@ jobs:
           libraries: ''
 
   online:
+    # Don't run online tests for tagged releases
+    if: |
+      github.event_name == 'pull_request' &&
```

Comment:
I am going to have to delegate checking this to @ConorMacBride but I think you just need the not a tag check?